agent/src/agent/cli/prompt/pipeline/__init__.py

```diff
@@ -27,4 +26,7 @@ def get_prompter(pipeline_: Pipeline, default_config: dict, advanced: bool) -> P
         source.TYPE_SOLARWINDS: SolarWindsPrompter,
         source.TYPE_ZABBIX: ZabbixPrompter,
     }
-    return prompters[pipeline_.source.type](pipeline_, default_config, advanced)
+    try:
+        return prompters[pipeline_.source.type](pipeline_, default_config, advanced)
+    except KeyError as e:
+        raise PipelineException(f'No prompter available for Pipeline with source type `{pipeline_.source.type}`') from e
```

Comment:
it should not be possible to call a source prompter that doesn't exist, they are filtered out in the _filter_without_cli, so I don't think we need to handle this exception. If a key error occurs, it's a sign for a developer something went wrong, maybe we can add a test if it doesn't exist